src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,77 @@
+package duke.task;
+
+/**
+ * Represents a Task to allow different types of tasks to inherit.
+ */
+public class Task {
+    /**
+     * Description of task.
+     */
+    protected String taskName;
+    /**
+     * Checks if task is completed or not.
+     */
+    protected boolean isDone;
+    /**
+     * '0' to represent incomplete, '1' to represent completed.
+     */
+    protected String completed;
```

Comment:
Hi, I had this extra string variable because it is used to convert the boolean to string which is to be placed into the text file when user starts running the app. Hope this clarifies!